config/subresource-integrity.php

```diff
@@ -52,4 +52,18 @@
     'hashes' => [
         // 'URL' => 'your_hash',
     ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Enabled Environments
+    |--------------------------------------------------------------------------
+    |
+    | These are the application environments enabled to generate hash. Don't
+    | include your local environment specially if you're using BrowserSync or
+    | any like it.
+    |
+    */
+
+    'enabled_env' => ['production'],
```

Comment:
As discussed in #48, I prefer to have an `enabled` config variable that will be either `true` or `false`.
To avoid breaking changes, it should default to `true`.

```suggestion
    'enabled' => env('SRI_ENABLED', true),
```